app/models/classification.rb

```diff
@@ -36,7 +37,16 @@ def self.scope_for(action, user, opts={})
     when :incomplete
       incomplete_for_user(user).includes(:subjects)
     when :project
-      joins(:project).merge(Project.scope_for(:update, user)).includes(:subjects)
+      return none if opts[:last_id] && !opts[:project_id]
```

Comment:
Did this because that's what unauthorized users are getting, but it deserves a better, less confusing response.